lib/rust/flexer/src/automata/alphabet_segmentation.rs

```diff
@@ -0,0 +1,82 @@
+//! Exports an alphabet for an arbitrary finite state automaton.
```

Comment:
leave the file named `alphabet` and the struct `Segmentation`. We are trying to use qualified imports everywhere, so you could then just write `use automata::alphabet` and use it like `t : alphabet::Segmentation` instead.